panel/panel.vala

```diff
@@ -952,22 +952,22 @@ public class Panel : Budgie.Toplevel
         }
         this.expanded = expanded;
         if (!expanded) {
-            if (wm_proxy != null) {
+            /*if (wm_proxy != null) {
```

Comment:
So my thoughts here is that, legitimately, our use of `GtkPopover` in budgie is the #1 thing breaking everything. I think the real solution is to stop `budgie-panel` using the window expansion system and switch to real top level windows